testcerts_test.go

```
@@ -19,6 +19,22 @@ func TestCertsUsage(t *testing.T) {
 		t.Errorf("Unexpected key length from public/private key")
 	}
 
+	t.Run("Verify Cert",
+		func(t *testing.T) {
+			ca := NewCA()
+			cert := ca.Cert()
+			if cert == nil {
+				t.Errorf("Expected certificate, got nil")
+			}
+			if cert.SerialNumber.Cmp(big.NewInt(42)) != 0 {
```

golangci-lint (1.62.2)

SA5011: possible nil pointer dereference

(staticcheck)